test-versions.txt

```diff
@@ -1,6 +1,9 @@
 # Well known working crio tag/commit/branch
 crio_version=400713a58bedb88678e84b72d4620847c8d27fec
 
+# Runc version compatible with CRI-O
```

Comment:
Great! But to be clearer, should this say, "Runc version compatible with 'crio_version'" ?

Reply:
Agree, fixed.